tqdm/std.py

```diff
@@ -1005,7 +1009,7 @@ def __init__(self, iterable=None, desc=None, total=None, leave=True,
         # Store the arguments
         self.iterable = iterable
         self.desc = desc or ''
-        self.total = total
+        self.total = total + initial if total else total
```

Comment:
this seems dangerous as a user would not expect their explicit `total` to be overridden by this.

Reply:
basically see https://github.com/tqdm/tqdm/issues/689#issuecomment-651259944

Reply:
I gotcha, however, this will prevent a progress bar from appearing unless a total is explicitly defined when using initial.  Possible fix is to add more logic to check if the user explicitly defined total before overriding.  Lmk if you want to implement this.

Reply:
I think it's best to wait and see if anyone requests that feature. For now I think it's safe to assume

- most people using `initial` will also specify `total`
- most people trying to use `initial` without a `total` will quickly try adding `total` (e.g. https://github.com/tqdm/tqdm/issues/689#issue-416724068)